llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -40,14 +40,13 @@ static void AssertRelease(bool Cond, const char *Msg) {
 
 StringRef getStringLiteralArg(const CallInst *CI, unsigned ArgNo,
                               SmallVectorImpl<Instruction *> &DelInsts,
-                              GlobalVariable *&DelGlob) {
+                              GlobalVariable *&SymGlob) {
   Value *V = CI->getArgOperand(ArgNo)->stripPointerCasts();
 
   if (auto *L = dyn_cast<LoadInst>(V)) {
     // Must be a
     // vvvvvvvvvvvvvvvvvvvv
-    // @.str = private unnamed_addr constant[18 x i8]
-    //   c"_ZTS11MyBoolConst\00", align 1
+    // @.str = private unnamed_addr constant[10 x i8] c"SpecConst\00", align 1
```

Comment:
This change makes the following comment with LLVM IR invalid for types.

Reply:
ok